Project.toml

```diff
@@ -25,6 +25,7 @@ ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 GPUArrays = "0c68f7d7-f131-5f86-a1c3-88cf8149b2d7"
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 HypothesisTests = "09f84164-cd44-5f33-b23f-e6b0d136a0d5"
+IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
```

Comment:
IJulia shouldn't be in here, causes problems in project-specific envs. The containers will have it anyway.